resources/elbv2_load_balancers.go

```diff
@@ -219,3 +300,47 @@ func fetchElbv2LoadBalancerAvailabilityZoneAddresses(ctx context.Context, meta s
 	res <- p.LoadBalancerAddresses
 	return nil
 }
+
+type lbAttributes struct {
```

Comment:
are there any "remain" attributes ?

Reply:
According to the documentation there shouldn't be any.

Reply:
okay